Resources/Prototypes/_ES/Catalog/Fills/wardrobe_fun.yml

```
@@ -0,0 +1,117 @@
+- type: entity
+  parent: ESWardrobeTheatre
+  id: ESWardrobeTheatreFilled
+  suffix: Filled
+  components:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:NestedSelector
+        tableId: ESFillWardrobeTheatre
+        rolls: !type:RangeNumberSelector
+          range: 3, 5
+
+- type: entityTable
+  id: ESFillWardrobeTheatre
+  table: !type:GroupSelector
+    children:
```

```suggestion
  table: 
    group:
```
you can do this across the whole file